core/templates/pages/release-coordinator-page/components/cancel-beam-job-dialog.component.ts

```diff
@@ -0,0 +1,51 @@
+// Copyright 2021 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Component for managing an Apache Beam job.
+ */
+
+import { Component, Inject } from '@angular/core';
+import { MatDialog, MatDialogRef, MAT_DIALOG_DATA } from '@angular/material/dialog';
+import { catchError, take } from 'rxjs/operators';
+
+import { BeamJobRun } from 'domain/admin/beam-job-run.model';
+import { ReleaseCoordinatorBackendApiService } from 'pages/release-coordinator-page/services/release-coordinator-backend-api.service';
+import { AlertDialogComponent } from 'pages/release-coordinator-page/components/alert-dialog.component';
+
+@Component({
+  selector: 'cancel-beam-job-dialog',
+  templateUrl: './cancel-beam-job-dialog.component.html',
+})
+export class CancelBeamJobDialogComponent {
```

Comment:
Ditto as with alert.

Reply:
I think this component has special responsibilities that would need a new component regardless of our choice, so there isn't much we can reuse.